backend/src/main/java/com/ambacam/rest/actions/ActionResource.java

```diff
@@ -0,0 +1,51 @@
+package com.ambacam.rest.actions;
+
+import com.ambacam.model.Action;
+import com.ambacam.model.Role;
+import com.ambacam.rest.ApiConstants;
+import com.ambacam.service.ActionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@RestController
+@RequestMapping(ApiConstants.ACTION_ITEM)
```

Comment:
Please could you add the annotation `@Validated` here? Although I do not really see the importance of that, but better add it first and then we will discuss about if you find it necessary or not.

Reply:
Alright